org.python.pydev.updatesite/site.xml

```diff
@@ -3,10 +3,7 @@
    <description name="PyDev Update Site" url="https://raw.githubusercontent.com/ponder-lab/Pydev/pydev_9_3_1/org.python.pydev.updatesite">
       PyDev Updatesite
    </description>
-   <feature url="features/org.python.pydev.feature_9.3.0.202203051235.jar" id="org.python.pydev.feature" version="9.3.0.202203051235">
-      <category name="org.python.pydev.updatesite"/>
-   </feature>
-   <feature url="features/org.python.pydev.feature.source_9.3.0.202203051235.jar" id="org.python.pydev.feature.source" version="9.3.0.202203051235">
```

Comment:
Why is the "source" feature removed?

Reply:
Since I added other directories to the source for the feature. The only feature that was coming up for the feature was this one which is the one from the feature.xml. I believe the "source" feature is used to get the source information in eclipse. For example, being able to use the `Open Call Hierarchy`

Reply:
If there is no reason to remove it, please put it back.

Reply:
I see that the target shows up with the feature source, but it is not coming up as an option for the features in Eclipse, so I can't build and obtain the feature.source. The way the create the source is through tycho-source-feature-plugin (see features/org.python.pydev.feature/pom.xml)

Reply:
We can see that from the mvn build (https://github.com/ponder-lab/Pydev/actions/runs/3425641863/jobs/5706626077#step:11:9602) it is actually building the source-jar but in my eclipse it is not, I am trying to figure out how to fix.

Reply:
9

Reply:
That's interesting.